### DISCUSSION TITLE
Keyball44反応しないキーについて

### DISCUSSION THREAD

**Discussion by UsernameIsHere** [Yowkees/keyball#385] [Q&A] [answered]
ライト13の下にあるキーだけ反応しない
スイッチを換えたり、ピンセットでテストしても反応しないので
たぶんダイオードが基板のどちらかに問題があると思う
まずはダイオード交換から始めたいが
対応しているダイオードがどれなのかがわからない
教えて頂けると嬉しいです🙏

![WhatsApp Image 2023-10-09 at 12 02 51 (1)](https://github.com/Yowkees/keyball/assets/51040990/d836350a-f1ac-4851-8acf-656a7c88d4ad)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

ご連絡いただいたキーに対応するダイオードは青丸のものになります。
たしかに半田が球状なのでテンプラしているかも知れませんね。
併せて、キーソケット自体の半田付けも見直していただければ復活できると思います。

お手数ですがご対応お願いいたします。

![KB44_keyfix4](https://github.com/Yowkees/keyball/assets/84209772/ee1206e1-f185-4979-b17a-705e569005cb)
